regression/cbmc/array-function-parameters/test.c

```diff
@@ -0,0 +1,37 @@
+#include <assert.h>
+#include <stdlib.h>
+
+typedef struct Node_T Node_T;
```

Comment:
Nit pick: C-land doesn't have camels.

Reply:
I'll replace with the cprover naming convention